monster_waves/RandomGenerator.cpp

```diff
@@ -0,0 +1,26 @@
+#include "RandomGenerator.hpp"
+
+RandomNumber::RandomNumber()
+{
+    std::random_device rd;
```

Comment:
@JakubMastalski można to śmiało przenieść do listy inicjalizacyjnej i zrobić w jednej linijce nie dwóch. Jest to ciut bardziej wydajne.
[ ] - przenieść operację ustawiania ziarna do listy inicjalizacyjnej konstruktora